src/components/logIn/FormLogIn.jsx

```diff
@@ -0,0 +1,175 @@
+import * as Style from "../styles/styleSheet.js";
+import users from "../../mockUsers/users";
+import youtubeApi from "../../utils/Youtube";
+
+const FormLogIn = () => {
+  const [errorUser, setErrorUser] = useState("");
+  const [errorUserNotFound, setErrorUserNotFound] = useState("");
+  const [user, setUser] = useState("");
+  const [password, setPassword] = useState("");
+  const [errorPassword, setErrorPassword] = useState("");
+  const { userSession, setUserSession } = useContext(VideoContext);
+  const { changinggSetChangingg } = useContext(VideoContext);
+  const { setFavoritesFlag } = useContext(VideoContext);
+  const { setVideoMetaInfo } = useContext(VideoContext);
+
+  var userMap = new Map();
+
+  const onChange = (e) => {
+    setUser(e.target.value);
+    //setUserSessionLocal(e.target.value);
+  };
+  const onChange2 = (e) => {
+    setPassword(e.target.value);
+    //setUserPassLocal(e.target.value);
+  };
+
+  const validate = () => {
+    let isValid = true;
+    if (!user) {
+      setErrorUser("User required");
+      isValid = false;
+    } else if (user.length < 3) {
+      setErrorUser("User length higher than 3");
+      isValid = false;
+    } else {
+      setErrorUser("");
+    }
+    if (!password) {
+      setErrorPassword("Pass required");
+      isValid = false;
+    } else {
+      setErrorPassword("");
+    }
+
+    return isValid;
+  };
+
+  const validateUser = () => {
+    let isValid = true;
+    if (validate()) {
+      if (userMap.get(user)) {
+        if (userMap.get(user).pass !== password) {
+          isValid = false;
+          setErrorUserNotFound("Wrong Pass");
+        }
+      } else {
+        isValid = false;
+        setErrorUserNotFound("User Not Found");
+      }
+    } else {
+      isValid = false;
+    }
+    return isValid;
+  };
+
+  const getmapUsers = () => {
+    let localusers = { ...users };
+
+    for (let i in localusers) {
+      userMap.set(localusers[i].user, localusers[i]);
+    }
+  };
+
+  const videoTendencies = async () => {
+    console.log("do request with params FORMLOIn");
+    const params = {
+      q: ""
+    };
+
+    const varResponse = await youtubeApi.get("/search", {
+      params
+    });
+
+    changinggSetChangingg(false);
+    setFavoritesFlag(false);
+
+    if (
+      varResponse &&
+      varResponse.data &&
+      Array.isArray(varResponse.data.items)
+    ) {
+      setVideoMetaInfo(varResponse.data.items);
+    }
+  };
+
+  const onLogIn = () => {
+    getmapUsers();
+    if (validateUser()) {
+      setUserSession({
+        user: user,
+        pass: password,
+        email: "",
+        loggedIn: true
+      });
+    }
+    videoTendencies();
+  };
+  const onLogInWithoutSession = () => {
+    setUserSession({
+      user: "noSession",
+      pass: "",
+      email: "",
+      loggedIn: true
+    });
+    videoTendencies();
+  };
+
+  return (
+    <Style.FormWrapper>
+      <div>
+        <Style.Input1
```

Comment:
Same comment, use Input1 instead of Style.Input1 by exporting just the required styled components